### DISCUSSION TITLE
Module not found error while doing next start

### DISCUSSION THREAD

**Discussion by Swarupa567** [vercel/community#3314] [Help] [answered]
### Page to Investigate

-

### Steps to Reproduce

1. Created a new repo with NextJS and Typescript
2. Created a reusuable component and imported in APP.js
3. made a build using next build
4. to render a web page while doing next start, following error occurred as module not found.
5. 
<img width="776" alt="image" src="https://github.com/vercel/community/assets/22514881/1f6fdfee-94d7-4c4b-98ce-378ab3802baf">

**Comment by swarnava** [comment, marked as answer]
Hi,

Can you remove `.next` folder > rerun `npm run build && npm start` and try? Also are you using any custom config in `next.config.js` or webpack?

**Comment by Swarupa567** [comment]
Yes, I removed the .next folder and it resolved by running npm run build.
